CefSharp.Core/Internals/StringUtils.cpp

```diff
@@ -78,5 +78,20 @@ namespace CefSharp
                 cef_string_copy(pStr, str->Length, &cefStr);
             }
         }
+
+        [DebuggerStepThrough]
+        String^ StringUtils::CreateExceptionString(Exception^ ex, int limit)
```

Comment:
No need for the code duplication, can just go with the `C#` version in a common place.